docs/about.md

```diff
@@ -0,0 +1,21 @@
+# About KLoadGen
+
+KLoadGen is a plugin for JMeter which acts as a load generator designed to send Kafka messages and that supports both AVRON and JSON structures. 
```

Comment:
Please, fix the typo, is AVRO instead of AVRON. JSON Schema instead of only JSON. 
Do you mind add also PROTOBUF?
Cheers

Reply:
Done!